src/BenchmarksApps/Mvc/benchmarks.mvcjson.yml

```diff
@@ -182,3 +182,17 @@ scenarios:
         presetHeaders: 'jsonInput'
         path: /EchoTodo
         bodyFile: https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/assets/todo.json
+
+  # MapAction EchoForm
+  MapActionEchoTodoForm:
+    application:
+      job: mapaction
+    load:
+      job: bombardier
+      variables:
+        verb: POST
+        headers:
```

Comment:
There should be a `customHeaders` variable that is free-form, but the way you did this works too. It's just redifining the set of reusable headers and point to this new key.

Reply:
Good point -- I ended up simplifying this further and removing the dependency on the external body file.